web/views/user_view.ex

```diff
@@ -20,13 +20,18 @@ defmodule CodeCorps.UserView do
   has_many :user_skills, serializer: CodeCorps.UserSkillView
 
   def photo_large_url(user, _conn) do
+    IO.inspect user
     CodeCorps.UserPhoto.url({user.photo, user}, :large)
   end
 
   def photo_thumb_url(user, _conn) do
     CodeCorps.UserPhoto.url({user.photo, user}, :thumb)
   end
 
+  def name(user, _conn) do
+    user.first_name <> " " <> user.last_name
```

Comment:
I would rethink this to not add in the empty space `" "` in the event that either `first_name` or `last_name` are blank. You can probably do this via pattern matching on the `name` function.